SymantecCloudSOC: fix date format error (#34640)
demisto/content: Packs/SymantecCloudSOC/ReleaseNotes/1_0_8.md

#### Integrations

##### Symantec CloudSOC Event Collector

> - Fixed an issue where the **fetch-events** command failed to fetch events due to wrong date format.

Reviewer: ```suggestion
- Fixed an issue where the **fetch-events** command failed to fetch events when fetched events have an unexpected date format.
```
